inertia/utils.py

```diff
@@ -13,23 +13,22 @@ def model_to_dict(model):
 
 
 class InertiaJsonEncoder(DjangoJSONEncoder):
-    def default(self, value):
-        if hasattr(value.__class__, "InertiaMeta"):
+    def default(self, o):
```

Comment:
This is probably a better name given that's what the parent object uses, but this isn't really related to typing. Why not type this function since it's being modified?

Reply:
Good catch, pyright was complaining exactly about the name being different from the base class. I think it's a good idea to add typing there as well, I'll change it

Reply:
I suppose that is coming up very soon. I'm all for dropping 3.9, it just feels like something that should be dropped in a bigger release. 
Perhaps we could merge this after another PR to drop 3.9? Or we could temporarily just install django-stubs with the type checker tool when doing type checking, but that isN,t super clean :/